app/components/avo/paginator_component.html.erb

```diff
@@ -22,7 +22,10 @@
     </div>
   </div>
   <div class="flex">
-    <div class="flex-2 sm:flex sm:items-center sm:justify-between space-y-2 sm:space-y-0 text-center sm:text-left">
+    <%= content_tag :div, class: class_names("flex-2 sm:flex sm:items-center sm:justify-between space-y-2 sm:space-y-0 text-center sm:text-left", {
+      "pagy-gem-version-7": pagy_major_version <= 7,
+      "pagy-gem-version-8": pagy_major_version >= 8,
+    }) do %>
```

Comment:
```suggestion
    <%= content_tag :div, class: "flex-2 sm:flex sm:items-center sm:justify-between space-y-2 sm:space-y-0 text-center sm:text-left pagy-gem-version-#{pagy_major_version}" do %>
```

IMO interpolation fits better on this use case increasing readability and should reduce the computing time

Reply:
I thought about that, but it will break when pagy 9 will launch.
If pagy 9 will have the same markup, then, it will continue to work.

Reply:
Yup, makes sense